.github/workflows/release_parallel.yml

```diff
@@ -0,0 +1,218 @@
+name: Parallel Release
+on:
+    workflow_call:
+        secrets:
+            GPG_SIGNING_KEY:
+                required: true
+            GH_TOKEN:
+                required: true
+            NPM_TOKEN:
+                required: true
+            LASTMJS_GITHUB_TOKEN:
+                required: true
+
+jobs:
+    prepare-release:
+        name: Prepare Release
+        runs-on: ubuntu-latest
+        env:
+            GPG_SIGNING_KEY: ${{ secrets.GPG_SIGNING_KEY }} # All commits must be verified
+            GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        outputs:
+            release-version: ${{ steps.get-version.outputs.release-version }}
+            test-infos: ${{ steps.get-test-infos.outputs.test-infos }}
+        steps:
+            - uses: actions/checkout@v4
+              with:
+                  ref: ${{ github.event.pull_request.head.ref || github.ref }}
```

Comment:
Why the ors?

Reply:
same as with the token, it's been like that the whole time. Questioning the old ways of authenticating or checking out was not in the scope of this issue. I thought it best, for time reasons, to only question the old ways of spawning jobs.

Reply:
open an issue to explore this for 1.0 milestone

Reply:
https://github.com/demergent-labs/azle/issues/2200